src/assets/src/utils.ts

```diff
@@ -19,22 +20,13 @@ export const redirectToSearch = (term: string) => {
     location.href = `/search/${term}/?redirected=true`;
 }
 
-const invalidUniqnameMessage = (uniqname: string) =>
-    uniqname + " is not a valid user. Please make sure the uniqname is correct, and that they have logged onto Remote Office Hours Queue at least once."
-
-export const validateAndFetchUser = async (uniqname: string) => {
-    if (uniqname.length < 3) throw new Error("Uniqnames must be at least 3 characters long.");
-    if (uniqname.length > 8) throw new Error("Uniqnames must be at most 8 characters long.");
-    if (!uniqname.match(/^[a-z]+$/i)) throw new Error("Uniqnames cannot contain non-alphanumeric characters.");
+export const confirmUserExists = async (uniqname: string) => {
```

Comment:
What do you think about moving this to `validate.ts`? You are *validating* the existence of a user.

Reply:
Yeah, it could live somewhere else. Sure, I'll move it there for now. I guess it just felt like a fundamentally different kind of validation (using an API endpoint and depending on dynamic data, rather than a deterministic function check).

Reply:
I moved this to `validation`.